src/app/movies/movies.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="row">
   <mat-form-field appearance="fill">
     <mat-label>Link filmu</mat-label>
-    <input matInput [(ngModel)]="movie.link">
+    <input matInput [(ngModel)]="movie.id">
```

Comment:
nie zapinaj tego inputu do property filmu, no chyba że to tymczasowe rozwiązanie - docelowo user wklei nie id tylko cały link, a po wklejeniu na jakieś ngChange inputu będziesz musiał link przeparsować na obiekt filmu